### PR TITLE
add test

### PR DESCRIPTION
**Pull request by chudesya**
Approve my PR, please :)

### CODE REVIEW

**Review by NataliaKubiak — APPROVED** [review]

**Comment by chudesya** [comment]
Approve my PR, please :) https://github.com/RedRoverSchool/JenkinsQA_06/pull/1111/

File: src/test/java/school/redrover/AnaBelGroupTest.java [hunk 1]
```diff
@@ -36,9 +36,14 @@ public void testAddDescription() {
 
         Assert.assertEquals(verify.getText(), "testDesctiprion1");
     }
-
+
```
Comment (ainuramergalimova) [inline src/test/java/school/redrover/AnaBelGroupTest.java:39]: Why ...

Comment (chudesya) [reply src/test/java/school/redrover/AnaBelGroupTest.java:39]: Because

**Review by ainuramergalimova — APPROVED** [review]
Good job!

**Pull request merged**